app/webroot/js/misp2.4.50.js

```diff
@@ -2489,3 +2494,23 @@ $(".queryPopover").click(function() {
 	});
 });
 
+function requestAPIAccess() {
+	var destination = 'users';
+	var action = 'request_API';
+	url = "/" + destination + "/" + action + "/";
```

Comment:
fair enough. though I'm not a big fan of hardcoding, but I guess in this case.. Though maybe we should have only a cpl of functions who do async req to the server and get the url, container to update, extra functions to perform as params.